agent/src/agent/data_extractor/observium/observium.py

```diff
@@ -31,18 +31,21 @@ def _get(pipeline_: Pipeline):
 def _create_metrics(data: dict, pipeline_: Pipeline) -> list:
     metrics = []
     # these values must be outside the for loop for optimization purposes
-    fields = field.build_fields(pipeline_.dimension_configurations)
-    value_paths = pipeline_.value_paths
-    value_paths = dict(zip(value_paths, value_paths))
+    fields_dims = field.build_fields(pipeline_.dimension_configurations)
+    fields_meas = field.build_fields(pipeline_.values_configurations)
```

Comment:
Maybe we should call it `measurement_configurations` instead of value? Need to think about it more

Reply:
Named it `value_configurations` because there is `value` in config. Maybe we should rename both? `measurements` and `measurement_configurations`?

Reply:
we cannot rename both because it will break backward compatibility. I think `measurement_configurations` will be better because it describes how to create an Anodot measurement, and `value` refers only to the field in the source data that contains a value